### DISCUSSION TITLE
UE failed to setup/configure GW interface

### DISCUSSION THREAD

**Discussion by initChat** [srsran/srsRAN_Project#337] [General Help] [answered]
I follow this tutorial: O-RAN NearRT-RIC and xApp https://docs.srsran.com/projects/project/en/latest/tutorials/source/flexric/source/index.html

I used to successfully running it (open5gs -> nearRT-RIC -> gnb -> ue -> ping), until my ubuntu encountered problem and need to reset and installed everything again.

I finish installed and running  
- 5gc (open5gs docker from srsRAN_Project)
- near-RT RIC
- gNB (from srsRAN_Project)

But I encounter problem when running srsRAN_4G, the PDU session cannot be established
```
Active RF plugins: libsrsran_rf_zmq.so
Inactive RF plugins: 
Reading configuration file ./ue_zmq.conf...

Built in Release mode using commit eea87b1d8 on branch master.

Opening 1 channels in RF device=zmq with args=tx_port=tcp://127.0.0.1:2001,rx_port=tcp://127.0.0.1:2000,base_srate=11.52e6
Supported RF device list: zmq file
CHx base_srate=11.52e6
Current sample rate is 1.92 MHz with a base rate of 11.52 MHz (x6 decimation)
CH0 rx_port=tcp://127.0.0.1:2000
CH0 tx_port=tcp://127.0.0.1:2001
Current sample rate is 11.52 MHz with a base rate of 11.52 MHz (x1 decimation)
Current sample rate is 11.52 MHz with a base rate of 11.52 MHz (x1 decimation)
Waiting PHY to initialize ... done!
Attaching UE...
Random Access Transmission: prach_occasion=0, preamble_index=0, ra-rnti=0x39, tti=334
Random Access Complete.     c-rnti=0x4601, ta=0
RRC Connected
Failed to setup/configure GW interface
RRC NR reconfiguration successful.
```
My srsRAN_4G during cmake show zeromq disable by default 
```
-- Checking for module 'ZeroMQ'
--   No package 'ZeroMQ' found
-- Found libZEROMQ: /usr/include, /usr/lib/x86_64-linux-gnu/libzmq.so
```
I try delete czmq and install again, and delete srsRAN_4G and install again but still encounter problem. Please help me. I attach log here.

[gnb.txt](https://github.com/srsran/srsRAN_Project/files/13472136/gnb.txt)
[nearRT_RIC.txt](https://github.com/srsran/srsRAN_Project/files/13472137/nearRT_RIC.txt)
[open5gs.txt](https://github.com/srsran/srsRAN_Project/files/13472138/open5gs.txt)
[ue.txt](https://github.com/srsran/srsRAN_Project/files/13472139/ue.txt)

**Comment by pgawlowicz** [comment, marked as answer]
you need to run:
```
sudo ip netns add ue1
```

**Reply by initChat** [reply]
Thank you, I forgot to add it before run the UE, I try to add it after and maybe that's why I could not successfully set PDU session. Thank you.
